packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -383,8 +376,8 @@ class FlameChartItem {
     });
   }
 
-  static const defaultTextColor = Color(0xFF000000);
-  static const selectedTextColor = Color(0xFFFFFFFF);
+  final defaultTextColor = isDarkTheme ? Colors.white : Colors.black;
```

Comment:
You might be able to use the default themed foreground and background colors here.